cmd/web.go

```diff
@@ -71,7 +71,7 @@ and it takes care of all the other things for you`,
 }
 
 // newMacaron initializes Macaron instance.
-func newMacaron() *macaron.Macaron {
+func NewMacaron() *macaron.Macaron {
```

Comment:
Can we move this (and `RegisterRoutes` to a new `modules/router` instead? Since they really shouldn't be here anyway :)